ZipBuilder/Sources/ZipBuilder/LaunchArgs.swift

```diff
@@ -199,6 +205,30 @@ struct LaunchArgs {
       currentReleasePath = nil
     }
 
+    // Parse the zipPods key.
+    if let zipPodsPath = defaults.string(forKey: Key.zipPods.rawValue) {
+      let url = URL(fileURLWithPath: zipPodsPath)
+      guard fileChecker.fileExists(atPath: url.path) else {
+        LaunchArgs.exitWithUsageAndLog("Could not parse \(Key.releasingSDKs) key: value passed " +
+          "in is not a file URL or the file does not exist. Value: \(zipPodsPath)")
+      }
+      let jsonData: Data
+      do {
+        jsonData = try Data(contentsOf: url)
+      } catch {
+        fatalError("Could not read JSON file at \(url). \(error)")
+      }
+      // Get pods, with optional version, from the JSON file.
+      let decoder = JSONDecoder()
+      do {
+        zipPods = try decoder.decode([CocoaPodUtils.VersionedPod].self, from: jsonData)
+      } catch {
+        fatalError("Could not parse JSON manifest at \(url). \(error)")
+      }
```

Comment:
I think it's reasonable to put these in the same `do/catch` block since they're both parsing the JSON manifest.

Reply:
Much more readable. Thanks!